src/inc/corinfo.h

```diff
@@ -834,13 +834,14 @@ enum CorInfoFlag
     CORINFO_FLG_AGGRESSIVE_OPT        = 0x01000000, // The method may contain hot code and should be aggressively optimized if possible
     CORINFO_FLG_DISABLE_TIER0_FOR_LOOPS = 0x02000000, // Indicates that tier 0 JIT should not be used for a method that contains a loop
     CORINFO_FLG_NOSECURITYWRAP        = 0x04000000, // The method requires no security checks
+    CORINFO_FLG_REQUIRES_JUMP_STAMP   = 0x08000000, // This IL method requires a Jump Stamp for Versionablity
```

Comment:
We have `CORJIT_FLAG_PROF_REJIT_NOPS` already. Why do we need another flag that does the same?

Reply:
Removed